app/templates/plugins

```diff
@@ -1,6 +1,4 @@
 # List of all plugin urls
 
-https://downloads.wordpress.org/plugin/advanced-custom-fields.zip
-https://downloads.wordpress.org/plugin/acf-field-date-time-picker.zip
```

Comment:
So ACF already has a date/time picker built in to the plugin [here](https://www.advancedcustomfields.com/resources/date-time-picker/). Which is why I removed it from the list.